client/src/annoMatrix/viewCreators.ts

```diff
@@ -62,8 +78,7 @@ export function clip(annoMatrix, qmin, qmax) {
 Private utility functions below
 */
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'mask' implicitly has an 'any' type.
-function _maskToList(mask) {
+function _maskToList(mask: Uint8Array): OffsetArray | null {
   /* convert masks to lists - method wastes space, but is fast */
   if (!mask) {
```

Comment:
given that mask can't be null (ie, type of param disallows it), you can remove this check and remove null from the type signature.  That will remove the upstream error suppression.

alternatively, you could change the return signature, and still do the falsy check on this line (likely requires a cast)

Reply:
Removed [null check](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/viewCreators.ts#L80) and corresponding [suppression](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/viewCreators.ts#L23).